modzy/jobs.py

```diff
@@ -447,7 +453,25 @@ def submit_files_bulk(self, model, version, sources, explain=False):
             raise
         return Job(open_job, self._api_client)
 
-    def submit_aws_s3(self, model, version, source, access_key_id, secret_access_key, region, explain=False, source_name='job'):
+    def _append_input(self, job, input_item_name, data_item_name, input_value, chunk_size):
+        iterable = None
+        if isinstance(input_value, (bytes, bytearray)):
+            iterable = bytes_to_chunks(input_value, chunk_size)
+        else:
+            iterable = file_to_chunks(input_value, chunk_size)
+            self.logger.debug(f"_append_input -> {iterable}")
+        i = 0
+        for chunk in iterable:
+            self.logger.debug(f"_append_input({job.job_identifier}, {input_item_name}, {data_item_name}, {type(input_value)}, {chunk_size}) chunk #{i}")
+            self._api_client.http.post(
```

Comment:
Within this PR you use both f-strings and format strings, it might be useful to use just one style consistently